internal/pkg/fixtures/configs/trans-with-shared-code-variables.json

```diff
@@ -1,6 +1,6 @@
 {
   "componentId": "keboola.variables",
-  "name": "Python Transformation Shared Code Variables",
+  "name": "Trans With Shared Code Variables",
```

Comment:
Urobil som trocha poriadok s nazvami a adresarmi, lebo som sa v tom stracal.

Reply:
🤔 A myslíš že zkrácení transformace na `Trans` tomu pomůže? 🙂

Reply:
Vysvetlim :slightly_smiling_face: 

**Povodny stav:**
![image](https://user-images.githubusercontent.com/19371734/139580351-eb64406e-74bc-4332-a18e-e81052a7a577.png)

Tj. mame v teste 2 python transformacie, jednou sa testuju samotne python transformacie, a 2. je tam pre `shared code + variables` (tj. na type az tak nezalezi).

Nazov adresara sa generuje z nazvu (ktory tu riesime).

Od toho zavisia aj nazvy ENV premennych v teste:
![image](https://user-images.githubusercontent.com/19371734/139580480-d1fc981a-432d-4c02-9412-52482a6783a6.png)
 
No a kedze su tam 2 configy, ktore zacinaju s `python_transfomation`, tak sa v tom blbo orientuje.

TJ: ci je tam skrata alebo nie ma vobec nenapadlo riesit, islo len o to aby nazov nezacinal rovnako.

**Skratku som nahradil**: https://github.com/keboola/keboola-as-code/pull/257/commits/30c831a385608c5e7ee3b310f4fb7e98abd8d2fe

Reply:
Aha, ok. 😃